front_end/html/course_admin.html

```diff
@@ -4,12 +4,28 @@ <h2>{{ course_basics["title"] }}</h2>
   <p>{{ course_details["introduction"] }}</p>
 
   {% if len(assignments) > 0 %}
+
+    <script>
+      var days = ['Sun','Mon','Tue','Wed','Thu','Fri','Sat'];
+      function convertDate(d) {
```

Comment:
Is this function the same as what is in course.html? If so, please make a common JavaScript file that contains functions used in multiple HTML files and source that rather than have two versions of the same function. That way if we change the logic, we won't need to remember to change it in two places.